yarn.lock

```diff
@@ -1161,6 +1161,13 @@ debug@^3.2.7:
   dependencies:
     ms "^2.1.1"
 
+debug@^4.3.3:
```

Comment:
You've got non flat dependencies here so you need to run `npx yarn-deduplicate` to fix that.

- [ ] Fix non flat deps